jax_triton/pallas/lowering.py

```diff
@@ -409,17 +410,34 @@ def _dot_general_lowering(ctx: TritonLoweringRuleContext, a, b, *,
                 allow_tf32=allow_tf32)
 triton_lowering_rules[jax.lax.dot_general_p] = _dot_general_lowering
 
-def _reduce_max_lowering(ctx: TritonLoweringRuleContext, a, *, axes):
-  assert len(axes) == 1
+def _reduce_lowering(triton_op, ctx: TritonLoweringRuleContext, a, *, axes):
+  if len(axes) != 1:
+    raise ValueError("`pallas` reduce operations only support one reduce axis.")
```

Comment:
Could loop over axes here right?

Reply:
I tried it but got the wrong numbers, I'm not sure it's actually supported (i.e. reducing a block twice).